source/utilities/formatters/index.js

```diff
@@ -63,9 +63,52 @@ const toPercentage = (value, decimal_points=2) => flow([
   addPostfix(" %")
 ])(value);
 
+/**
+  * Run a validation to know if the name of a variable is valid or not
+  * @example
+  * isVarNameValid("valid_var_name"); //=> true
+  * isVarNameValid("do"); //=> false
+  *
+  * @param {string} value - The suggested name of a variable
+  * @returns {boolean}
+  *
+  */
+
+const isVarNameValid = (value) => {
```

Comment:
This function will also be used in camelCaseToSnake function, but I'm not sure if this is a good place to put it, what do you think @Rantelo?

Reply:
The configuration in babel for "export default..." is missing